src/App.jsx

```diff
@@ -2,28 +2,23 @@
 // IMPORTS
 // ============================================================================================
 
-import { useContext } from "react";
-import { UserContext } from "./contexts/UserContext";
-
-import LoginPage from "./pages/LoginPage";
-import HomePage from "./pages/HomePage";
-
 import NavBar from "./components/NavBar";
 
 import "./App.css";
+import AppRoutes from "./AppRoutes";
 
 // ============================================================================================
 // COMPONENT DEFINITION
 // ============================================================================================
 
 function App() {
-  const { userInfo } = useContext(UserContext);
-
-  return <>
-    <NavBar />
-
-    {userInfo == null ? <LoginPage /> : <HomePage />}
-  </>;
+  return (
+    <>
+      <NavBar />
+      <AppRoutes />
+      {/* {userInfo == null ? <LoginPage /> : <HomePage />} */}
```

Comment:
Why you have commented this line out?

Reply:
This was part of the old process before the AppRoutes was created. I will delete it. Thanks for mentioning it